pkgs/tools/admin/amazon-ecr-credential-helper/default.nix

```diff
@@ -0,0 +1,21 @@
+{ buildGoPackage, fetchurl, lib, ... }:
+
+buildGoPackage rec {
+  name = "amazon-ecr-credential-helper-${version}";
+  version = "0.1.0";
+
+  goPackagePath = "github.com/awslabs/amazon-ecr-credential-helper";
+
+  src = fetchurl {
```

Comment:
Please use `fetchFromGitHub` instead of `fetchurl` because it relies on the hash of the contents of the repository instead of the hash of a tarball, and it would allow us to fix all GitHub src fetchers, if GitHub changes something about how it works.

Reply:
oh, thank you for pointing that out! I was always thinking about the time it takes to fetch the src.